kpi_dashboard/static/src/scss/kpi_dashboard.scss

```diff
@@ -0,0 +1,112 @@
+.o_dashboard_view {
+    height: 100%;
+    @include o-webclient-padding($top: $o-horizontal-padding/2, $bottom: $o-horizontal-padding/2);
+    display: flex;
+    >.gridster {
+        margin: 0 auto;
+        >ul {
+            >li {
+                text-align: center;
+                list-style: none outside none;
+            }
+        }
+    }
+    .updated_at {
```

Comment:
This class is not displayed correctly for lines without kpi. (title) I think that it should be hidden for non kpi lines.

Tested on runbot : 

![image](https://user-images.githubusercontent.com/3407482/79461057-f1488c00-7ff5-11ea-8c1d-b186c693fc09.png)

Do you have the same problem ? 

Trouble on Ubuntu 18.04 : Firefox 75.0 ; Chromium : 80.0.3987

Reply:
Someone made it contain a kpi Number 01. If you set no kpi_id it works correctly. Did it happen on runbot or in local (maybe demo data is wrong, but I see it right)

Reply:
on runbot. maybe data was corrupted.